src/wallet/asyncrpcoperation_sendmany.h

```diff
@@ -138,8 +139,8 @@ class TEST_FRIEND_AsyncRPCOperation_sendmany {
     
     // Delegated methods
     
-    void add_taddr_change_output_to_tx(CAmount amount) {
-        delegate->add_taddr_change_output_to_tx(amount);
+    void add_taddr_change_output_to_tx(CAmount amount, bool changeAddress=false) {
```

Comment:
```suggestion
    void add_taddr_change_output_to_tx(CAmount amount, bool changeAddress = false) {
```
nit, style/consistency